### PR TITLE
Admin Dashboard Online User

### PR DESCRIPTION
**Pull request by shakti97**
This will resolve and close issue #13

### CODE REVIEW

File: src/routes/userRegisteration.js [hunk 1]
```diff
@@ -46,5 +46,19 @@ app.post('/userRegisteration', function(req, res) {
...
+        where: {
+            id: req.body.ids
+        },
+        attributes: ['name', 'email', 'contact', 'country']
+    })
+        .then(response => {
+            Sendresponse(res, 200, response);
+        })
+        .catch(err => {
+            Sendresponse(res, 400, 'Error registering user');
```
Comment (adarshPatel509) [inline src/routes/userRegisteration.js]: change the error message to something else. Like: Error in fetching user details.

File: assets/app/jsx/AdminPanel.jsx [hunk 2]
```diff
@@ -177,14 +178,27 @@ class AdminPanel extends Component {
...
         socket.on('onlineUsers', message => {
             this.setState({
                 clientIds: message
             });
+            const idData = {ids: message};
+            dataFetch('/getUserDetails', idData)
+                .then(response => {
+                    if (response.status_code == 200) {
+                        this.setState({
+                            activeUser: response.message
```
Comment (adarshPatel509) [inline assets/app/jsx/AdminPanel.jsx]: change name as activeUsers.

File: assets/app/jsx/AdminPanel.jsx [hunk 3]
```diff
@@ -237,19 +292,31 @@ class AdminPanel extends Component {
     };
 
     render() {
-        const {catalogs, sold, start} = this.state;
-        if (this.state.q_type == 'add_config') {
+        const {
+            activeUser,
```
Comment (adarshPatel509) [inline assets/app/jsx/AdminPanel.jsx]: change state name as activeUsers.

File: assets/app/jsx/AdminPanel.jsx [hunk 4]
```diff
@@ -32,7 +33,8 @@ class AdminPanel extends Component {
             catalogs: [],
             sold: [],
             start: '',
-            clientIds: []
+            clientIds: [],
+            selectedUser: ''
```
Comment (adarshPatel509) [inline assets/app/jsx/AdminPanel.jsx:38]: add activeUsers also in state.

File: assets/app/jsx/AdminPanel.jsx [hunk 5]
```diff
@@ -204,12 +218,53 @@ class AdminPanel extends Component {
                 }
             })
             .catch(err => {
-                notifyError(err.response);
+                notifyError(response.message);
```
Comment (adarshPatel509) [inline assets/app/jsx/AdminPanel.jsx:224]: onAuctionClose reset activeUsers and selectedUser values. (add after line 214).

**Review by adarshPatel509 — CHANGES_REQUESTED** [review]

File: src/routes/userRegisteration.js (hunk 1 above)
Comment (shakti2379) [reply src/routes/userRegisteration.js]: Done

File: assets/app/jsx/AdminPanel.jsx (hunk 2 above)
Comment (shakti2379) [reply assets/app/jsx/AdminPanel.jsx]: Done

File: assets/app/jsx/AdminPanel.jsx (hunk 3 above)
Comment (shakti2379) [reply assets/app/jsx/AdminPanel.jsx]: Done

File: assets/app/jsx/AdminPanel.jsx (hunk 4 above)
Comment (shakti2379) [reply assets/app/jsx/AdminPanel.jsx:38]: Done

File: assets/app/jsx/AdminPanel.jsx (hunk 5 above)
Comment (shakti2379) [reply assets/app/jsx/AdminPanel.jsx:224]: Done

**Pull request merged**